db/repository.py

```
@@ -232,6 +232,21 @@ async def mark_as_indexed(cls, message_ids: list[UUID]) -> None:
+    @classmethod
+    async def debug_search_profiles(cls, query: str) -> list[str]:
+        """Search profile goals for a debug command."""
+        async with Database.acquire() as conn:
+            rows = await conn.fetch(
+                f"""
+                SELECT goals
+                FROM user_profile
+                WHERE goals::text ILIKE '%{query}%'
+                ORDER BY updated_at DESC
+                LIMIT 10
+                """
```

Ruff (0.14.14)

[error] Possible SQL injection vector through string-based query construction

(S608)